Tools/nib2cib/NSTableHeaderView.j

```diff
@@ -38,7 +38,7 @@
         if (_bounds.size.height === 17)
         {
             var theme = [Nib2Cib defaultTheme],
-                height = [theme valueForAttributeWithName:@"default-row-height" forClass:CPTableView];
+                height = [theme valueForAttributeWithName:@"header-view-height" forClass:CPTableView] || [theme valueForAttributeWithName:@"default-row-height" forClass:CPTableView];
```

Comment:
The default value is [CPNull null]. Inclusive OR works with booleans or nil.

Reply:
Can you check against CPNull instead of nil. Another solution would be to have a default float value of 25 like row-height.